### ISSUE TITLE
Aggregate syntax question

### ISSUE THREAD

**Issue by nolram88** [leangen/graphql-spqr#201]
I've been digging into how to implement aggregates for a single type in my project and here is an example I've found that seems pretty reasonable however I'm at a loss as to how to take the "fields" such as rating as an input for the "aggregate". Could I get a hint as to the approach for this?

```query {
  author (where: {id: {_eq: 1}}) {
    id
    name
    articles_aggregate {
      aggregate {
        count
        avg {
          rating
        }
        max {
          rating
        }
      }
      articles {
        id
        title
        rating
      }
    }
  }
}
```

**Comment by grrttedwards** [comment]
Take a look at the way Prisma (https://www.prisma.io/) handles this. 

TL;DR: Create fields such as `rating_count`, `rating_avg`, and `rating_max`, and treat them just like normal fields backed by resolver logic. GraphQL as a standard, and the tools built around it seem to want nothing to do with aggregation, I've found.
